packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -12,14 +12,33 @@
 			<NodeSettingsTabs v-model="openPanel" :nodeType="nodeType" :sessionId="sessionId" />
 		</div>
 		<div class="node-is-not-valid" v-if="node && !nodeValid">
-			<n8n-text>
+			<p :class="$style.warningIcon">
+				<font-awesome-icon icon="exclamation-triangle" />
+			</p>
+			<div class="missingNodeTitleContainer mt-s mb-xs">
+				<n8n-text
+					size="large"
+					bold
+				>
+					{{ $locale.baseText('nodeSettings.CommunityNodeUnknown.title') }}
+				</n8n-text>
+			</div>
+			<n8n-text v-if="isBaseNode">
```

Comment:
what about custom nodes? `CUSTOM.` types... we should check for community nodes and default to this